setup.cfg

```diff
@@ -92,6 +92,49 @@ stpipe.steps =
     jwst = jwst.stpipe.integration:get_steps
 pytest11 =
     report_crds_context = pytest_crds.plugin
+console_scripts =
+    asn_edit = jwst.scripts.asn_edit:main
+    asn_from_list = jwst.associations.asn_from_list:Main
+    asn_gather = jwst.scripts.asn_gather:main
+    asn_generate = jwst.associations.main:Main
+    asn_make_pool = jwst.scripts.asn_make_pool:main
+    assign_wcs = jwst.scripts.assign_wcs:main
+    collect_pipeline_cfgs = jwst.scripts.collect_pipeline_cfgs:main
+    coron = jwst.scripts.coron:main
+    create_data = jwst.scripts.create_data:main
+    csvconvert = jwst.csv_tools.csvconvert:CSVConvertScript
+    cube_build = jwst.scripts.cube_build:main
+    dark_current = jwst.scripts.dark_current:main
+    data_generate = jwst.scripts.data_generate:main
+    dqinit = jwst.scripts.dqinit:main
+    exp_to_source = jwst.exp_to_source.main:Main
+    flatfieldcorr = jwst.scripts.flatfieldcorr:main
+    fringecorr = jwst.scripts.fringecorr:main
+    ipc = jwst.scripts.ipc:main
+    jump = jwst.scripts.jump:main
+    linearitycorr = jwst.scripts.linearitycorr:main
+    make_header = jwst.scripts.make_header:main
+    migrate_data = jwst.scripts.migrate_data:main
+    minimum_deps = jwst.scripts.minimum_deps:write_minimum_requirements_file
+    move_wcs = jwst.scripts.move_wcs:main
+    okify_regtests = jwst.scripts.okify_regtests:main
+    outlier_detection = jwst.scripts.outlier_detection:main
+    persistencecorr = jwst.scripts.persistencecorr:main
+    photomcorr = jwst.scripts.photomcorr:main
+    pointing_summary = jwst.scripts.pointing_summary:main
+    rampfitcorr = jwst.scripts.rampfitcorr:main
+    refpix = jwst.scripts.refpix:main
+    resample = jwst.scripts.resample:main
+    saturationcorr = jwst.scripts.saturationcorr:main
+    schema_editor = jwst.scripts.schema_editor:main
+    schemadoc = jwst.scripts.schemadoc:main
+    set_telescope_pointing = jwst.scripts.set_telescope_pointing:main
```

Comment:
Though an appropriate change, one I agree with, the changes for both `set_telescope_pointing.py` and `set_velocity_aberration.py` to loose the `.py`, is a public API deprecation. Some smoother transition will be necessary. 

In particular, this (almost) cannot happen for the DMS Build 9.1; this will affect the rest of the automatic processing and it is a bit late in the development cycle to be foisting such a change on SDP.

Reply:
That's a good catch; I can add two more passthrough entry points and add warnings to them about future deprecation.

Reply:
done in 4bd4e96